config/initializers/rack_attack.rb

```diff
@@ -21,16 +21,13 @@ def headers
     ### Configure Cache ###
 
     # Note: The store is only used for throttling and fail2ban filtering;
-    # not blocklisting & safelisting. It must implement .increment and .write
-    # like ActiveSupport::Cache::Store
-
-    cache = Readthis::Cache.new(
+    # not blocklisting & safelisting
+    Rack::Attack.cache.store = ActiveSupport::Cache::RedisCacheStore.new(
+      namespace: 'rack-attack',
```

Comment:
the namespace should separate new data from old data, it means there will essentially be a reset on the rack-attack values after a deploy, but I think the timeouts are short-lived enough things will be okay?